configs/foreman/1.21.yaml

```diff
@@ -0,0 +1,383 @@
+---
+:project: foreman
+:github_org: theforeman
+:releases:
+  :1.21.0:
+    :redmine_version_id: 953
+:mash_scripts:
+  - foreman-1.21-rhel7-dist
+  - foreman-plugins-1.21-rhel7-dist
+  - foreman-client-1.21-rhel7
+  - foreman-client-1.21-rhel6
+  - foreman-client-1.21-rhel5
+  - foreman-client-1.21-fedora27
```

Comment:
I think f27 is EOL. Do we want to start supporting it?

Reply:
I feel like users tend to hold on for it